package.json

```diff
@@ -64,7 +64,7 @@
     "gl-scatter3d": "^1.0.4",
     "gl-select-box": "^1.0.1",
     "gl-spikes2d": "^1.0.1",
-    "gl-surface3d": "^1.2.0",
+    "gl-surface3d": "^1.2.2",
```

Comment:
the `png` files below should be in `./test/image/baselines/` not `./test/image/mocks/`